mobileapps/views.py

```diff
@@ -1149,7 +1157,7 @@ def __init__(self):
         self.permission_classes += (IsStaffView,)
 
     @transaction.atomic
-    def delete(self, request, theme_id, attribute):
+    def delete(self, _request, theme_id, attribute):
```

Comment:
Doesn't this view require any special permissions? Can anyone with API access make a request to this?

Reply:
There is `IsStaffOrReadOnlyView`. Please see [this line](https://github.com/edx-solutions/mobileapps-edx-platform-extensions/blob/9262213a3beee88ccbaeaccd2fe9bc087f5dcf42/mobileapps/views.py#L1029)

Reply:
I missed that.